contracts/admin/src/msg.rs

```diff
@@ -0,0 +1,42 @@
+use std::collections::HashMap;
+
+use serde::{Deserialize, Serialize};
+
+use finance::currency::SymbolOwned;
+use sdk::{
+    cosmwasm_std::Addr,
+    schemars::{self, JsonSchema},
+};
+
+use crate::common::{
+    type_defs::{MaybeMigrateGeneralContract, MigrateGeneralContracts, MigrateLpnContracts},
+    GeneralContracts, LpnContracts,
+};
+
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize, JsonSchema)]
+#[serde(rename_all = "snake_case", deny_unknown_fields)]
+pub struct InstantiateMsg {
+    pub general_contracts: GeneralContracts<Addr>,
+    pub lpn_contracts: HashMap<SymbolOwned, LpnContracts<Addr>>,
+}
+
+#[derive(Debug, Copy, Clone, Eq, PartialEq, Serialize, Deserialize, JsonSchema)]
+pub struct MigrateMsg {}
+
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize, JsonSchema)]
+#[serde(rename_all = "snake_case", deny_unknown_fields)]
+pub enum SudoMsg {
+    RegisterLpnContracts {
+        symbol: SymbolOwned,
+        contracts: LpnContracts<Addr>,
+    },
+    MigrateContracts(Box<MigrateContracts>),
```

Comment:
what's the purpose of `Box` here?

Reply:
Discussed in DMs. Will suppress linter warning.